.docker/images/nginx/tests/src/BlocksTest.php

```diff
@@ -7,17 +7,15 @@
 /**
  * Ensure that the blocks are respected by nginx.
  */
-class BlocksTest extends TestCase
```

Comment:
here and below - reformatted braces positions according to Drupal coding standards